flake.nix

```diff
@@ -0,0 +1,74 @@
+{
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
+    crane.url = "github:ipetkov/crane";
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+
+  outputs =
+    {
+      self,
+      nixpkgs,
+      crane,
+      flake-utils,
+      ...
+    }:
+    flake-utils.lib.eachDefaultSystem (
+      system:
+      let
+        pkgs = nixpkgs.legacyPackages.${system};
+
+        inherit (pkgs) lib;
+
+        craneLib = crane.mkLib pkgs;
+        src = craneLib.cleanCargoSource ./.;
+
+        commonArgs = {
+          inherit src;
+          strictDeps = true;
+          doCheck = false;
+
+          nativeBuildInputs = [
+            pkgs.pkg-config
+            pkgs.openssl.dev
+          ];
+
+          buildInputs = [
+            pkgs.pkg-config
+            pkgs.openssl
+          ];
+
+          BUILDIFIER_BIN = "${pkgs.bazel-buildtools}/bin/buildifier";
+        };
+
+        cargoArtifacts = craneLib.buildDepsOnly commonArgs;
+
+        pcb = craneLib.buildPackage (
+          commonArgs
+          // {
+            inherit cargoArtifacts;
+            pname = "pcb";
```

Comment:
When I build this I get a warning about Crane not detecting the name from Cargo.toml (which makes sense since it's just a workspace definition), I think it's coming from the `buildDepsOnly`.

Moving `pname = "pcb"` up to `commonArgs` seems to fix it I think

Reply:
will update